storage/applicationinsights/src/main/resources/ApplicationInsights.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+
+    Copyright 2017 The OpenZipkin Authors
+
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+    in compliance with the License. You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software distributed under the License
+    is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+    or implied. See the License for the specific language governing permissions and limitations under
+    the License.
+
+-->
+
+<ApplicationInsights xmlns="http://schemas.microsoft.com/ApplicationInsights/2013/Settings"
+    schemaVersion="2014-05-30">
+
+  <!-- HTTP request component (not required for bare API) -->
+
+  <TelemetryModules>
+    <Add
```

Comment:
since you only track telemetry manually - you do not need any auto-collection modules and initializers.